types/index.esm.d.ts

```diff
@@ -3175,10 +3173,7 @@ export const LogarithmicScale: ChartComponent & {
   new <O extends LogarithmicScaleOptions = LogarithmicScaleOptions>(cfg: AnyObject): LogarithmicScale<O>;
 };
 
-export type TimeScaleOptions = CartesianScaleOptions & {
-  min: string | number;
-  max: string | number;
```

Comment:
Either forgot to add these back or accidentally deleted them, after that looks fine

Reply:
Rushing and forgot to add them back.